alacritty_terminal/src/config/mod.rs

```diff
@@ -320,6 +218,10 @@ impl Percentage {
             value
         })
     }
+
+    pub fn raw_value(self) -> f32 {
```

Comment:
```suggestion
    pub fn inner(self) -> f32 {
```

Alternatively: `as_f32`.